client/src/plugins/vuetify.js

```diff
@@ -3,4 +3,19 @@ import Vuetify from 'vuetify/lib';
 
 Vue.use(Vuetify);
 
-export default new Vuetify({});
+export default new Vuetify({
+    theme: {
+        themes: {
+            light: {
```

Comment:
please, remove comments, the rest is fine, probably, themes better create as separate file, but it is not important. So removing comments would be enough, thx